src/main/java/seedu/address/logic/commands/edit/EditInternshipCommand.java

```diff
@@ -27,6 +27,7 @@
  * Edits an Internship Item in the resume book.
  */
 public class EditInternshipCommand extends EditCommand {
+    public static final String MESSAGE_FROM_TO_MISORDER = "\"from\" field cannot be after \"to\" field. ";
```

Comment:
a bit ambiguous, maybe try something like "from cannot be later than to"? cuz after could mean like after in the command